Sources/SwiftProtobufPluginLibrary/Descriptor.swift

```diff
@@ -791,7 +791,9 @@ public final class FieldDescriptor {
   static let kLastReservedNumber: Int = 19999
 
   /// Declared type of this field.
-  public let type: Google_Protobuf_FieldDescriptorProto.TypeEnum
+  public var type: Google_Protobuf_FieldDescriptorProto.TypeEnum { return _type }
+  private var _type: Google_Protobuf_FieldDescriptorProto.TypeEnum
```

Comment:
The more idiomatic expression here is:
```
public private(set) var type: Google_Protobuf_FieldDescriptorProto.TypeEnum
```

Reply:
I think I've mainly been using the private `_` version to help avoid accidentally setting things in any other place within the implementation. But either works.